engine/src/main/java/org/terasology/entitySystem/prefab/internal/PrefabFormat.java

```diff
@@ -54,7 +54,8 @@ public PrefabData load(ResourceUrn resourceUrn, List<AssetDataFile> inputs) thro
                 PrefabSerializer serializer = new PrefabSerializer(componentLibrary, typeSerializationLibrary);
                 return serializer.deserialize(prefabData);
             } else {
-                throw new IOException("Failed to read prefab for '" + resourceUrn + "'");
+                logger.error("Failed to read prefab for '" + resourceUrn + "'");
```

Comment:
Ok, I returned throwing exceptions, for correct work was enough catching in the _PostBeginSystem_